spec.html

```diff
@@ -26727,6 +26727,28 @@ <h1>Abstract Module Records</h1>
             </tr>
           </table>
         </emu-table>
+
+        <emu-clause id="sec-EvaluateModuleSync" type="abstract operation">
+          <h1>
+            EvaluateModuleSync (
+              _module_: a Module Record,
+            ): either a normal completion containing ~unused~ or a throw completion
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It synchronously evaluates _module_, provided that the caller guarantees that _module_'s evaluation will return an already settled promise.</dd>
+          </dl>
+
+          <emu-alg>
+            1. Assert: _module_ is not a Cyclic Module Record.
+            1. Let _promise_ be _module_.Evaluate().
+            1. Assert: _promise_.[[PromiseState]] is either ~fulfilled~ or ~rejected~.
+            1. If _promise_.[[PromiseState]] is ~rejected~, then
+              1. Perform HostPromiseRejectionTracker(_promise_, *"handle"*).
```

Comment:
```suggestion
              1. If _promise_.[[PromiseIsHandled]] is *false*, perform HostPromiseRejectionTracker(_promise_, *"handle"*).
              1. Set _promise_.[[PromiseIsHandled]] to *true*.
```

@tc39/ecma262-editors While writing the comment above, I realized that the logic in this PR is still not quite correct. The `HostPromiseRejectionTracker` hook is only meant to be called if the promise is not already handled in some other way (and given that this is a host-provided promise, it might be!), and we then need to mark the promise as handled.